lib/cover-project.nix

```diff
@@ -54,7 +54,7 @@ let
       (l: "${l}/share/hpc/vanilla/mix/${l.identifier.name}-${l.identifier.version}")
       libs;
 
-  srcDirs = map (l: l.src.outPath) libs;
+  srcDirs = map (l: l.src + l.srcSubDir or "") libs;
```

Comment:
This pattern happens enough that maybe we should have a function or an attribute on src that has it? `src.subDirPath` or something?

Reply:
We need it to work if someone overrides the `src` with a module.  In that case `l.src` may be a simple path.

Reply:
I guess it would have to be a function then. Kind of makes me want classes :p

Reply:
I think it would actually be ok.  The `src` is always the `root` from `rootAndSubDir` (unless `overrideAttrs` is used and I don't think we need to support that).  I'm not sure it should be `l.src.subDirPath` so I have added it to the passthru for components (so it is `l.srcSubDirPath`).